docs/sphinx_qtile.py

```diff
@@ -52,6 +55,27 @@
 
         Supported bar orientations: {{ obj.orientations }}
     {% endif %}
+    {% if is_widget and screen_shots %}
+    .. raw:: html
+
+        <table class="docutils">
+        <tr>
+        <td width="50%"><b>example</b></td>
+        <td width="50%"><b>config</td>
+        </tr>
+    {% for sshot, conf in screen_shots.items() %}
```

Comment:
The table has the image and config. Should we add a column for notes/comments? e.g. for `GroupBox` (screenshot in original post) it's not obvious that there's a window open in group 1.